README.md

```diff
@@ -1 +1,187 @@
-# countries-api
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>countries-api</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [countries-api] <a name="about-project"></a>
+
+**[countries-api]**is a web application that shows major regions and countries in the world. It fetches data from <a href="https://restcountries.com/#api-endpoints-v3-all">REST COUNTRIES API</a>
+
+<img src="./public/countries-api.png" width="150px">
+
+## 🛠 Built With <a name="built-with"></a>
+HTML, CSS, Es6 Javascript
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react-redux.js.org/">React Redux</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+>Here are the key features of the application.
+
+- Fetches data from the rest countries API
+- Displays a home page with different regions in the world
+- Allows users to click on each region to view detailed  information
+- Search bar to filter regions, countries by name
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://countries-api-8kc4.onrender.com/)
```

Comment:
- [ ] [REQUIRED] 👍  job deploying this project. However, the deployment link is broken. Kindly fix it before asking for another review.

<img width="671" alt="Screenshot 2023-01-14 at 14 33 59" src="https://user-images.githubusercontent.com/58692590/212475887-a1177d91-6146-42f1-9f86-12cda6606db0.png">